docs/integrations/dydx.mdx

```diff
@@ -9,42 +9,41 @@ title: dYdX
 
 **A:** Here is a quick (5-10 minutes) step-by-step breakdown. Please reach out to us at [skip.money/discord](http://skip.money/discord) if you have questions!
 
-#### **Download the Slinky binary.**
+#### **Download the Connect binary.**
 
-The best way to get the Slinky binary is in the GitHub releases page for Slinky.
+The best way to get the Connect binary is in the GitHub releases page for Connect.
 
 https://github.com/skip-mev/slinky/releases
 
-The **minimum** slinky version required for dYdX is [`v1.0.1`](https://github.com/skip-mev/slinky/releases/tag/v1.0.1)
+The **minimum** connect version required for dYdX is [`v1.0.1`](https://github.com/skip-mev/slinky/releases/tag/v1.0.1)
 
 We also provide a container image at [ghcr.io/skip-mev/slinky-sidecar](https://github.com/skip-mev/slinky/pkgs/container/slinky-sidecar/230279145?tag=v1.0.1)
 
-This will include the Slinky binary, `slinky`.
+This will include the Connect binary, `slinky`.
 
-#### **Integrate the Slinky sidecar into your setup.**
+#### **Integrate Connect into your setup.**
 
 The configuration of your validator setup requires you to tackle a few problems which we’ll mention here.
 
-**_Configure the Slinky process_**
+**_Configure the Connect process_**
 
-To run the slinky sidecar with stable defaults defined by the skip team
+To run connect with stable defaults defined by the skip team
```

Comment:
I would check with the team to see if there are still subproducts within Connect e.g. connect sidecar